### PR TITLE
Notifications page

### LINKED ISSUE #50: implement Notification page

### PR DESCRIPTION
**Pull request by LazakovichYauheni**
closed #50

### CODE REVIEW

File: ecommers/lib/ui/pages/notifications_page.dart [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+class NotificationsPage extends StatelessWidget {
+  static const _notificationDeviderIndent = 70.0;
+
+  static final DateFormat formatter = DateFormat('d MMM');
+
+  static final DateTime thirdDate = DateTime.utc(2020, 10, 15);
+  static final DateTime fourthDate = DateTime.utc(2020, 09, 20);
+
+  final String formattedThirdDate = formatter.format(thirdDate);
```
Comment (amarchenko25) [inline ecommers/lib/ui/pages/notifications_page.dart]: Please, remove unnecessary fields

File: ecommers/lib/ui/pages/notifications_page.dart [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  final String formattedFourthDate = formatter.format(fourthDate);
+
+  static String _getCorrectDateInString(DateTime date) {
+    if (date.day == DateTime.now().day) {
+      return DateFormat.jm().format(date).toString();
+    }
+    if (DateTime.now().month == date.month &&
+        (DateTime.now().day - date.day) >= 1) {
+      return 'Yesterday';
```
Comment (amarchenko25) [inline ecommers/lib/ui/pages/notifications_page.dart]: Please, move to the localization

File: ecommers/lib/ui/pages/notifications_page.dart [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+  static final DateFormat formatter = DateFormat('d MMM');
+
+  static final DateTime thirdDate = DateTime.utc(2020, 10, 15);
+  static final DateTime fourthDate = DateTime.utc(2020, 09, 20);
+
+  final String formattedThirdDate = formatter.format(thirdDate);
+  final String formattedFourthDate = formatter.format(fourthDate);
+
+  static String _getCorrectDateInString(DateTime date) {
```
Comment (amarchenko25) [inline ecommers/lib/ui/pages/notifications_page.dart]: consider renaming to _formatDate

**Comment by fryette** [comment]
fix conflicts plz

**Review by amarchenko25 — DISMISSED** [review]

**Review by amarchenko25 — APPROVED** [review]

**Pull request merged**